src/compressor/zlib/CMakeLists.txt

```diff
@@ -1,9 +1,31 @@
 # zlib
 
-set(zlib_sources
-  CompressionPluginZlib.cc
-  CompressionZlib.cc
-)
+if(INTEL_SSE4_1)
+	set(zlib_sources
+	  CompressionPluginZlib.cc
+	  ZlibCompressor.cc
+	  ${CMAKE_SOURCE_DIR}/src/isa-l/igzip/igzip.c
+	  ${CMAKE_SOURCE_DIR}/src/isa-l/igzip/hufftables_c.c
+	  ${CMAKE_SOURCE_DIR}/src/isa-l/igzip/crc_utils_01.asm
+	  ${CMAKE_SOURCE_DIR}/src/isa-l/igzip/crc_utils_04.asm
+	  ${CMAKE_SOURCE_DIR}/src/isa-l/igzip/igzip_body_01.asm
+	  ${CMAKE_SOURCE_DIR}/src/isa-l/igzip/igzip_body_04.asm
+	  ${CMAKE_SOURCE_DIR}/src/isa-l/igzip/igzip_finish.asm
+	  ${CMAKE_SOURCE_DIR}/src/isa-l/igzip/igzip_stateless_01.asm
+	  ${CMAKE_SOURCE_DIR}/src/isa-l/igzip/igzip_stateless_04.asm
+	  ${CMAKE_SOURCE_DIR}/src/isa-l/igzip/crc_data.asm
+	  ${CMAKE_SOURCE_DIR}/src/isa-l/igzip/crc32_gzip.asm
+	  ${CMAKE_SOURCE_DIR}/src/isa-l/igzip/detect_repeated_char.asm
+	  ${CMAKE_SOURCE_DIR}/src/isa-l/igzip/igzip_multibinary.asm
+	  ${CMAKE_SOURCE_DIR}/src/isa-l/igzip/igzip_stateless_base.c
+	  ${CMAKE_SOURCE_DIR}/src/isa-l/igzip/igzip_base.c
+	)
+else(INTEL_SSE4_1)
+	set(zlib_sources
+	  CompressionPluginZlib.cc
+	  ZlibCompressor.cc
+	)
```

Comment:
If INTEL_SSE4_1 is found I'm getting errors saying header files weren't found (missing -I flags). This can be fixed by adding:
`
    target_include_directories(ceph_zlib PRIVATE "${CMAKE_SOURCE_DIR}/src/isa-l/include")
`
under the target_link_libraries for ceph_zlib on line 32.